java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2823,22 +2823,26 @@ void testScanSum() {
     try (ColumnVector v1 = ColumnVector.fromBoxedInts(1, 2, null, 3, 5, 8, 10)) {
       // Due to https://github.com/rapidsai/cudf/issues/8462 NullPolicy.INCLUDE
       // tests have been disabled
-//      try (ColumnVector result = v1.scan(Aggregation.sum(), ScanType.INCLUSIVE, NullPolicy.INCLUDE);
-//           ColumnVector expected = ColumnVector.fromBoxedInts(1, 3, null, null, null, null, null)) {
-//        assertColumnsAreEqual(expected, result);
-//      }
-
-      try (ColumnVector result = v1.scan(Aggregation.sum(), ScanType.INCLUSIVE, NullPolicy.EXCLUDE);
+    // try (ColumnVector result = ColumnVector.scan(v1, Aggregation.sum(),
```

Comment:
nit: could you please revert the format changes for the comments. It is a bit confusing to see what is a real change and what is not.

I also looks like because https://github.com/rapidsai/cudf/issues/8462 was closed that you updated some of the tests, but didn't remove the commented out code. If you did update a test could you remove the corresponding commented out code and if you updated all of the tests could you remove the comment about issue 8462?